esp/esp/program/modules/handlers/jsondatamodule.py

```diff
@@ -762,6 +763,15 @@ def student_max_count(clslist):
             }
             dictOut["stats"].append({"id": "splashinfo", "data": splashinfo_data})
         
+        #   Add accounting stats
+        lt = ProgramAccountingController(prog).default_payments_lineitemtype()
```

Comment:
Can you use a more descriptive variable name?

Also, this is the second place where this code is written. Can you add a method on ``ProgramAccountingController`` that returns the tuple ``(num_payments, total_payments)``?